models/reviews-model.js

```diff
@@ -54,9 +54,17 @@ exports.fetchReviewById = (review_id = 1) => {
 
 exports.fetchAllReviews = () => {
 	return db
-		.query(`SELECT * FROM reviews ORDER BY created_at DESC`)
-		.then((reviews) => {
-			console.log(reviews.rows);
+		.query(
+			`SELECT reviews.created_at, reviews.title, reviews.category, reviews.review_id, 
```

Comment:
You could save yourself some typing here and make it much dry-er with reviews.* - which will select every column beginning with 'reviews.'